cfgov/jinja2/v1/_includes/molecules/global-eyebrow.html

```diff
@@ -23,45 +23,46 @@
                 <span class="m-global-eyebrow_tagline-usa">United States Government</span>
             </div>
             <div class="m-global-eyebrow_actions">
-                <ul class="list__unstyled
-                           list__horizontal
+                <ul class="m-list
+                           m-list__unstyled
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/3092/commits/cc9f53bf3265003a4ce4ce82a604fccfebc0ede9